one/src/lib.rs

```diff
@@ -167,6 +169,24 @@ struct DaemonOpts {
             env = "CERAMIC_ONE_CORS_ALLOW_ORIGINS"
         )]
     cors_allow_origins: Vec<String>,
+
+    /// Enable experimental feature flags
+    #[arg(
+        long,
+        value_parser = parse_flags::<ExperimentalFeatureFlags>,
```

Comment:
Why the custom parser? It seems easier to just derive `ValueEnum` on the two types and do something like this, which gives us nicer help text/matching.

```rust
    #[arg(
        long,
        use_value_delimiter = true,
        value_delimiter = ',',
        env = "CERAMIC_ONE_EXPERIMENTAL_FEATURE_FLAGS"
    )]```
    
plus I don't think this quite works how it's expected as it's requiring all the values to be passed

Reply:
Yeah, I was trying to get it to list the enum values, but that ended up not working how I wanted. Switching to value_delimiter